python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -103,6 +103,49 @@ def __get__(self, obj, cls=None):
             raise AttributeError()
 
 
+def Timestamp_Timedelta__new__(cls, *args, **kwargs):
+    # Call fast/slow constructor
+    # This takes care of running __init__ as well, but must be paired
+    # with a removal of the defaulted __init__ that
+    # make_final_proxy_type provides.
+    if len(args) > 0 and args[0] is pd.NaT:
```

Comment:
Is this specifically so an `pd.NaT` arg gets returned? For example, `pd.Timestamp("NaT")` also returns `pd.NaT`

Reply:
Right, this isn't need for this PR.